boards/shields/nrf21540_ek/doc/index.rst

```diff
@@ -0,0 +1,84 @@
+.. _nrf21540_ek:
+
+nRF21540 EK
+###########
+
+The nRF21540 EK (Evaluation Kit) is an RF front-end module (FEM) for
+Bluetooth Low Energy, Bluetooth mesh, 2.4 GHz proprietary, Thread, and
+Zigbee range extension.
+When combined with an nRF52 or nRF53 Series SoC, the nRF21540 RF FEM’s
++21 dBm TX output power and 13 dB RX gain ensure a superior link budget for
+up to 16x range extension.
+
+The nRF21540 complementary device has a 50 Ω SMA input and 2x 50 Ω SMA outputs.
```

Comment:
I'm not sure about the input/output naming here, since we are receiving RF signals through output ports :slightly_smiling_face:

Reply:
What about ?
 The nRF21540 complementary device has a 50 Ω SMA transceiver interface and 2x 50 Ω SMA antenna interfaces.

Reply:
Done